mmv1/products/compute/Network.yaml

```diff
@@ -123,10 +123,16 @@ properties:
     immutable: true
     validation:
       function: 'verify.ValidateGCEName'
+  - name: 'networkId'
+    description: |
+      The unique identifier for the resource. This identifier is defined by the server.
+    api_name: id
+    output: true
   - name: 'numericId'
     type: String
     description: |
-      The unique identifier for the resource. This identifier is defined by the server.
+      (Deprecated) The unique identifier for the resource. This identifier is defined by the server.
```

Comment:
Can you add a `deprecation_message`  on the field following https://googlecloudplatform.github.io/magic-modules/breaking-changes/make-a-breaking-change/#field-deprecation-due-to-removal-or-rename

Reply:
Thanks - updated both places